src/box/relay.cc

```diff
@@ -188,6 +188,14 @@ struct relay {
 	double txn_lag;
 	/** Relay sync state. */
 	enum relay_state state;
+	/** Whether relay should speed up the next heartbeat dispatch. */
+	bool need_new_vclock_sync;
```

Comment:
Nit: `vclock_sync_is_pending` would probably be a better name, but I don't insist - up to you.

Reply:
I'd rather leave the current name